spectrum/checks.py

```diff
@@ -349,6 +349,9 @@ def annotations(self, profile_id, access='public'):
     def digests(self):
         return self._list_api('/digests', 'digest')
 
+    def digest(self, id):
+        return self._item_api('/digests/%s' % id, 'digest')
```

Comment:
I'm not sure if a `GET` will work here if the content is `"stage": "preview"` in the digests API without providing an `Authorization` header. Temporarily, I think the `PUT` to the endpoint is not working just yet.

Reply:
Can't see it from here, but the instance of `ApiCheck` I am using has the `Authorization` header passed in the constructor.